server/protocols/jmap/src/test/java/org/apache/james/jmap/utils/JsoupHtmlTextExtractorTest.java

```diff
@@ -71,6 +71,64 @@ public void toPlainTextShouldUpperCaseLabels() {
     }
 
     @Test
+    public void toPlainTextShouldHandleListsWell() {
+        String html = "<ul>Here is my awesome list:" +
+            "  <li>JMAP</li>" +
+            "  <li>IMAP</li>" +
+            "</ul>" +
+            "<p>Followed with some text</p>" +
+            "<p>And some other text</p>";
+        String expectedPlainText = "Here is my awesome list:  \n" +
+            " - JMAP  \n" +
+            " - IMAP\n" +
+            "\n" +
+            "Followed with some text\n" +
+            "\n" +
+            "And some other text\n" +
+            "\n";
+        assertThat(textExtractor.toPlainText(html)).isEqualTo(expectedPlainText);
+    }
+
+    @Test
+    public void tableShouldBeWellHandled() {
+        String html = " <table style=\"width:100%\">\n" +
```

Comment:
The thing is you put `\n` in html just to see it pretty in your editor here you do not need it to have a new line because of the java construct 'lineA' +
'lineB'
So I agree with @quynhn and I would remove them